content/cucumber/state.md

```diff
@@ -191,5 +262,88 @@ See the [`spring-txn`](https://github.com/cucumber/cucumber-jvm/tree/master/exam
 
 If you're using a [browser automation](/browser-automation/) tool that talks to your application over HTTP the transactional approach
 will not work if your [step definitions](/cucumber/#step-definitions) and the web application serving HTTP request each have their own database connection.
+If you're using a [browser automation](/browser-automation/) tool that talks to your application over HTTP, the
+transactional approach will not work if your [step definitions](/cucumber/#step-definitions) and the web application serving
+HTTP request each have their own database connection.
+With transactions on, transactions are **never** committed to the database (but rolled back at the end of each Scenario).
+Therefore, the web server's connection will never see data from Cucumber, and therefore your browser won't either.
+Likewise, Cucumber's connection won't see data from the web server.
+
+In this case, you will have to turn off database transactions and make sure the data is explicitly deleted before each Scenario.
+
+## Ruby on Rails
+
+### Turn of transactions
+If you're using [Ruby on Rails](/tools/#ruby-on-rails) it's easy to turn off transactions for a feature or particular scenarios. Just use the `@no-txn` tag, like this:
```

Comment:
`Just use` => `Use`

Reply:
Yes; done (and removed some other instances of `just`)